src/Appwrite/Utopia/Response/Model/User.php

```diff
@@ -47,12 +47,24 @@ public function __construct()
                 'default' => '',
                 'example' => 'john@appwrite.io',
             ])
+            ->addRule('phone', [
+                'type' => self::TYPE_STRING,
+                'description' => 'User phone number.',
```

Comment:
We should mention the format to expect with link to relevant information about it.